content/docs/setup/kubernetes/quick-start-ibm/index.md

```diff
@@ -61,69 +39,95 @@ This guide installs the current release version of Istio.
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Add the IBM Cloud Helm repository to your Helm instance:
+### Deploy the Istio Helm charts
+
+1. Install the `istio-init` chart to bootstrap all the Istio's CRDs:
 
     {{< text bash >}}
-    $ helm repo add ibm-charts https://registry.bluemix.net/helm/ibm-charts
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
     {{< /text >}}
 
-### Deploy the Istio Helm chart
-
-1. If using a Helm version prior to 2.10.0, install Istio’s Custom Resource Definitions via `kubectl apply`, and wait a few seconds for the CRDs to be committed
-to the Kubernetes API server:
+    To verify all Istio's CRDs were committed in the Kubernetes api-server, check the number of CRDs with the specified suffix with the following command. Verify that the number of total CRDs created was `56` for Istio:
 
     {{< text bash >}}
-    $ kubectl apply -f https://raw.githubusercontent.com/IBM/charts/master/stable/ibm-istio/templates/crds.yaml
+    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
```

Comment:
You can display the expected output, 56, here:
```
    {{< text bash >}}
    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
    56
    {{< /text >}}
```

Reply:
Done and updated to 58.

Reply:
Done.